src/table/Table.js

```diff
@@ -0,0 +1,255 @@
+  height: t.maybe(t.Number),
+  /** the desired width of the table */
+  width: t.maybe(t.Number),
+  /** if true, the table will fit the container if width and height */
+  autoSize: t.maybe(t.Boolean),
+  /** if true, the columns will split evenly the width, avoiding horizontal scrollbars */
+  autoSizeColumns: t.maybe(t.Boolean),
+  /** height in pixel of every row */
+  rowHeight: t.maybe(t.Number),
+  /** height in pixel of header */
+  headerHeight: t.maybe(t.Number),
+  /** height in pixel of groupHeader */
+  groupHeaderHeight: t.maybe(t.Number),
+  /** height in pixel of footer */
+  footerHeight: t.maybe(t.Number),
+  /** method to get the desired row of data */
+  rowGetter: t.Function,
+  /** number of row displayed in the table */
+  rowsCount: t.maybe(t.Number),
+  cellRenderer: t.Function,
+  children: t.ReactNode,
+  sort: t.maybe(t.struct({
+    by: t.maybe(t.String),
+    dir: t.maybe(t.enums.of(['asc', 'desc'], 'sortDir'))
+  })),
+
+  selectedRows: t.maybe(t.list(t.Num)),
+  onRowSelect: t.maybe(t.Func),
+  onRowsSelect: t.maybe(t.Func),
+  selectionType: t.enums.of(['none', 'single', 'multi'], 'selectionType'),
+
+  className: t.maybe(t.String),
+  style: t.maybe(t.Object),
+  id: t.maybe(t.String)
+}), isInvariantSatisfied, 'FixedDataTableWrapperProps');
+
+@pure
+@skinnable()
+@props(Props)
+export default class Table extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      height: this.props.height,
+      width: this.props.width,
+      scrollToRow: this.props.selectedRows[0]
+    };
+  }
+
+  static defaultProps = {
+    width: 400,
+    height: 400,
+    selectedRows: []
+  };
+
+  componentDidMount() {
+    this.node = React.findDOMNode(this.refs.wrapper);
+    if (this.props.selectionType !== 'none') {
+      this.attachKeyPressListener();
+      this.scrollToSelectedRow();
+    }
+    if (this.props.autoSize) {
+      this.autoSize();
+      this.startAutoSizeInterval();
+    }
+  }
+
+  componentDidUpdate(newProps) {
+    if (newProps.selectedRows[0] !== this.props.selectedRows[0]) {
+      this.scrollToSelectedRow();
+    }
+  }
+
+  componentWillUnmount() {
+    this.removeKeyPressListener();
+    this.stopAutoSizeInterval();
+  }
+
+  startAutoSizeInterval = () => {
+    this.autoSizeInterval = setInterval(this.autoSize, 200);
+  };
+
+  stopAutoSizeInterval = () => {
+    clearInterval(this.autoSizeInterval);
+  };
+
+  autoSize = () => {
+    const { height: currentHeight, width: currentWidth } = this.state;
+    const { clientHeight: height, clientWidth: width } = this.node;
+    if ((height !== currentHeight) || (width !== currentWidth)) {
+      this.setState({ height, width }, this.scrollToSelectedRow);
+    }
+  };
+
+  scrollToSelectedRow = () => {
+    this.setState({
+      scrollToRow: this.props.selectedRows[0]
+    });
+  };
+
+  onScrollStart = () => {
+    this.setState({
+      scrollToRow: null
+    });
+  };
+
+  getDirectionFromKeyCode = (keyCode) => {
```

Comment:
This has changed because `FixedDataTableWrapper` now supports `Shortcuts`. I think this is out of the scope of the `Table` component for now.